addons/account/models/chart_template.py

```diff
@@ -241,7 +241,8 @@ def _load(self, sale_tax_rate, purchase_tax_rate, company):
             })
 
         # Set the transfer account on the company
-        company.transfer_account_id = self.env['account.account'].search([('code', '=like', self.transfer_account_code_prefix + '%')])[:1]
+        company.transfer_account_id = self.env['account.account'].search([(
+            'code', '=like', self.transfer_account_code_prefix + '%', 'company_id', '=', company.id)])
```

Comment:
@zaoral This code can not be right.  I agree it is better to be sure to take the transfer account of the proper company however.